pkg/users/account-create-dialog.js

```diff
@@ -244,22 +278,43 @@ export function account_create_dialog(accounts, shells) {
 
         errs.user_name = validate_username(user_name, accounts);
 
-        return password_quality(password, force)
-                .catch(ex => {
-                    errs.password = (ex.message || ex.toString()).replaceAll("\n", " ");
-                })
+        if (!state.home_dir)
+            errs.home_dir = _("No home directory specified");
+
```

Comment:
It could just let `useradd` use the default directory, and not specify a custom one. This is the "95%" case anyway. create() already DTRT and doesn't specify the `-d` option if the home dir is falsy.

Reply:
Yep, if someone clears the input, it should fall back to default. It's not a required field in the command line, and it doesn't have to be here.

Same for any other optional field.